massa-versioning-worker/Cargo.toml

```diff
@@ -11,15 +11,20 @@ parking_lot = "0.12"
 thiserror = "1.0"
 num_enum = "0.5"
 nom = "7.1"
+tokio = { version = "1.23", features = ["full"] }
+tracing = "0.1"
+queues = "1.1.0"
 
 # custom module
 massa_time = { path = "../massa-time" }
 massa_models = { path = "../massa-models" }
 massa_serialization = { path = "../massa-serialization" }
+massa_versioning_exports = { path = "../massa-versioning-exports" }
 
 [dev-dependencies]
 chrono = "0.4"
 more-asserts = "0.3"
+tokio = { version = "1.23", features = ["full"] }
```

Comment:
Any reason to have tokio as deps and dev-deps?

Reply:
None, this is an error introduced during a rebase. Fixed in c7799ada571784ede621718057dfee8291fcda94